slices/ca-certificates.yaml

```diff
@@ -4,30 +4,12 @@ essential:
   - ca-certificates_copyright
 
 slices:
-  # This slice offers a standard chunk of the ca-certificates package.
-  # It contains the default certificates at /usr/share/ca-certificates/mozilla,
-  # a certificates bundle at /etc/ssl/certs/ca-certificates.crt and a config
-  # file at /etc/ca-certificates.conf.
-  data:
-    essential:
-      - ca-certificates_data-no-certs
-    contents:
-      # This config file is generated by the deb's maintainer scripts.
-      /etc/ca-certificates.conf: {text: FIXME, mutable: true}
-      /usr/share/ca-certificates/mozilla/**:
-    mutate: |
-      certs_dir = "/usr/share/ca-certificates/mozilla/"
-      names = [
-        "mozilla/{}".format(name) for name in content.list(certs_dir)
-      ]
-      content.write("/etc/ca-certificates.conf", "\n".join(names))
-
   # This slice only contains the certificates bundle at /etc/ssl/certs/ca-certificates.crt.
   # To keep the individual certificates at /usr/share/ca-certificates/mozilla/ or the
-  # configuration file at /etc/ca-certificates.conf, please use the "data" slice.
+  # configuration file at /etc/ca-certificates.conf, please use the "data-with-certs" slice.
   # Please note that the mutation script (partially) mimics the deb's maintainer
   # scripts to generate the certificates bundle.
-  data-no-certs:
```

Comment:
please keep this slice. Now that we've introduced it, we have to keep it otherwise it will break those who are already using it. just make

```yaml
data-no-certs:
  essential:
    - ca-certificates_data
```

Reply:
Fixed.